File: DynamicTables.Api/ClientApp/src/app/admin/entitytypefield/IEntityTypeField.ts
import { IEntityBase } from '../../shared/IEntityBase';
import { EnumFieldType } from './EnumFieldType';

export interface IEntityTypeField extends IEntityBase {
  icon: string;
  label: string;
  html: string;
  javaScript: string;
  css: string;
  code: string;
  fotmat: string;
  fieldType: EnumFieldType;
}

File: DynamicTables.Core/Interfaces/Entities/IEntityTypeField.cs
using System;
using DynamicTables.Core.Entities.Enums;

namespace DynamicTables.Core.Interfaces.Entities
{
    public interface IEntityTypeField : IEntityBase
    {
        string Label { get; set; }

        string Html { get; set; }

        string JavaScript { get; set; }

        string Code { get; set; }

        string Format { get; set; }

        EnumFieldType FieldType { get; set; }
    }
}
